java/connector-node/risingwave-sink-dynamic/risingwave-sink-dynamic-http-sink/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.risingwave</groupId>
+        <artifactId>risingwave-sink-dynamic</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.example</groupId>
+    <artifactId>risingwave-sink-dynamic-http-sink</artifactId>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.example</groupId>
+            <artifactId>risingwave-sink-dynamic-api</artifactId>
+            <version>0.1.0-SNAPSHOT</version>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/com.getindata/flink-http-connector -->
+        <dependency>
+            <groupId>com.getindata</groupId>
+            <artifactId>flink-http-connector</artifactId>
+            <version>0.10.0</version>
+        </dependency>
+        <dependency>
```

Comment:
We can't move flink-connector-base to mock-runtime module, because we HttpDynamicTableSinkFactory extend AsyncDynamicTableSinkFactory, And AsyncDynamicTableSinkFactory is in flink-connector-base